articles/customizing-django-admin/index.md

```diff
@@ -0,0 +1,245 @@
+
+class  Author(models.Model):
+  gender_male  =  "Male"
+  gender_female  =  "Female"
+  gender_other  =  "Other"
+  name  =  models.CharField(max_length=100)
+  gender  =  models.CharField(max_length=100,
+      choices=(
+      (gender_female, gender_female),
+      (gender_male, gender_male),
+      (gender_other, gender_other)
+      )
+      )
+  country  =  models.CharField(max_length=100)
+
+  def  __str__(self):
+      return  self.name
+
+class  Details(models.Model):
+  book_name  =  models.CharField(max_length=100)
+  category = models.ForeignKey(Category, on_delete=CASCADE)
+  pages = models.IntegerField(default=1)
+  publisher  =  models.ForeignKey(Publisher, on_delete=models.CASCADE)
+  Author  =  models.ForeignKey(Author, on_delete=CASCADE)
+
+  def  __str__(self):
+      return  self.book_name
+```
+We can now go ahead and register our models in `admin.py` by adding the following lines of code.
+```python
+from django.db import models
+from .models import Category, Publisher ,Details, Author
+
+class  categoryAdmin(admin.ModelAdmin):
+   pass
+
+class  publisherAdmin(admin.ModelAdmin):
+   pass
+
+class  detailsAdmin(admin.ModelAdmin):
+   pass
+
+class  authorAdmin(admin.ModelAdmin):
+   pass
+
+admin.site.register(Category, categoryAdmin)
+admin.site.register(Publisher, publisherAdmin)
+admin.site.register(Details, detailsAdmin)
+admin.site.register(Author, authorAdmin)  
+```
+Then make the migrations and migrate using the below commands.
+
+```bash
+$ py manage.py makemigrations
+$ py manage.py migrate
+```
+Now, let us create a superuser in order to start customizing our admin site.
+```bash
+$ py manage.py createsuperuser
+```
+Now run the development server using the command below:
+```bash
+$ py manage.py runserver
+```
+You can now log in using the superuser credentials at http://127.0.0.1:8000/admin/.
+Now you might want to go ahead and add some input in the fields we created.
+###  1 -  Setting plural text for models
+We can change how the models *category* and *details* are appearing in plural at the admin site by adding the below code to `models.py`.
+```python
+class  Category(models.Model):
+# ...
+  class  Meta:  #new
+      verbose_name_plural  =  "Categories"     
+
+class  Details(models.Model):
+# ...
+  class  Meta:   #new
+      verbose_name_plural  =  "Details"
+```
+###  2. Changing the Django administration header text
+To change the admin site header-text,login page, and the HTML title tag to our bookstore's instead, add the following code in `urls.py`.
+```python
+#...
+admin.site.site_header  =  "Custom bookstore admin"  
+admin.site.site_title  =  "Custom bookstore admin site"
+admin.site.index_title  =  "Custom Bookstore Admin"
+```
+*site_header* changes the *Django administration* text which appears on the login page. The *site_title* appears on the admin page after we have logged in. The *index_title* changes our site's title that appears on the top of the page.
+
```

Comment:
The `site_title` is does not appear on the admin page as you have indicated above. Please see [this](https://docs.djangoproject.com/en/dev/ref/contrib/admin/#adminsite-attributes) document and fix your explanations. Also, write the 'site_title', etc as inline code blocks. ie. \`site_title`